### DISCUSSION TITLE
Why the dockers state is restarting？

### DISCUSSION THREAD

**Discussion by yangxuan21** [langgenius/dify#10634] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

When I deploy using a container, docker.api_1
The status of docker_standbox_1 and dockerworker_1 has always been Restarting. Where did this go wrong?

```
sudo docker-compose ps
       Name                      Command                  State                                         Ports
------------------------------------------------------------------------------------------------------------------------------------------------
docker_api_1          /bin/bash /entrypoint.sh         Restarting
docker_db_1           docker-entrypoint.sh postg ...   Up (healthy)   5432/tcp
docker_nginx_1        sh -c cp /docker-entrypoin ...   Up             0.0.0.0:444->443/tcp,:::444->443/tcp, 0.0.0.0:7002->80/tcp,:::7002->80/tcp
docker_redis_1        docker-entrypoint.sh redis ...   Up (healthy)   6379/tcp
docker_sandbox_1      /main                            Restarting
docker_ssrf_proxy_1   sh -c cp /docker-entrypoin ...   Up             3128/tcp
docker_weaviate_1     /bin/weaviate --host 0.0.0 ...   Up
docker_web_1          /bin/sh ./entrypoint.sh          Up             3000/tcp
docker_worker_1       /bin/bash /entrypoint.sh         Restarting

```

**Comment by lelouch-lampeski** [comment]
Have you solved that?I have the saame problem.The nginx and ssrf_proxy images are restarting all the time.I've checked the log,but I can't colve that.I can't access the dify homepage before the problem solved. 
![微信图片_20241126142951](https://github.com/user-attachments/assets/ff550279-f28f-4d65-a80b-dd6d3a712330)
![微信图片_20241126142956](https://github.com/user-attachments/assets/32205b1c-b663-443a-b10d-1807d6d1f6ab)

**Reply by yangxuan21** [reply]
It needs to modify Nginx configuration, container IP is incorrect.

**Reply by lelouch-lampeski** [reply]
Could you please explain it in more detail? I tried to work it out the way you said but found I couldn't understand it at my level.....

**Reply by yangxuan21** [reply]
Sorry, I didn't look carefully earlier.I did not make the error  as in the second picture. My mistake was that the docker is always been Restarting, but it was ultimately resolved by configuring IP in Nginx.

**Reply by JUshenghb** [reply]
Can yon solve it?bro!!!wo have same problems!My docker.api_1、docker_standbox_1 and dockerworker_1 always Restarting!!!

**Comment by JUshenghb** [comment]
Can yon solve it?bro!!!wo have same problems!My docker.api_1、docker_standbox_1 and dockerworker_1 always Restarting!!!

**Reply by seasurt** [reply]
Me too, did you solve it?
